xbmc/utils/ActorProtocol.h

```diff
@@ -54,10 +55,18 @@ class Message
   Message() {isSync = false; data = NULL; event = NULL; replyMessage = NULL;};
 };
 
+class MessageHandle : public ::KODI::UTILS::CScopeGuard<Message*, nullptr, void(Message*)>
```

Comment:
As discussed on Slack: Comment added, `MessageHandle` moved to Wayland implementation